sp800-63-3/sec5_DIRM.md

```diff
@@ -27,7 +27,7 @@ As such, agencies SHALL assess the risk of proofing, authentication, and federat
 
 [Section 5.3](#section5-3) provides impact categories specific to digital identity to assist in the overall application of the RMF.
 
-Risk assessments determine the extent to which risk must be mitigated by the identity proofing, authentiation, and federation processes. These determinations drive the relevant choices of applicable technologies and mitigation strategies, rather than the desire for any given technology driving risk determinations. Once an agency has completed the overall risk assessment; selected individual assurance levels for identity proofing, authentication, and federation (if applicable); and determined the processes and technologies they will will employ to meet each assurance level, agencies SHALL develop a "Digital Identity Acceptance Statement", in accordance with NIST 800-53r5 IA-1 a.1. See [Section 5.5.](#daps) for more detail on the necessary content of the Digital Identity Acceptance Statement.
+Risk assessments determine the extent to which risk must be mitigated by the identity proofing, authentication, and federation processes. These determinations drive the relevant choices of applicable technologies and mitigation strategies, rather than the desire for any given technology driving risk determinations. Once an agency has completed the overall risk assessment; selected individual assurance levels for identity proofing, authentication, and federation (if applicable); and determined the processes and technologies they will will employ to meet each assurance level, agencies SHALL develop a "Digital Identity Acceptance Statement", in accordance with NIST 800-53r5 IA-1 a.1. See [Section 5.5.](#daps) for more detail on the necessary content of the Digital Identity Acceptance Statement.
```

Comment:
Should include citation to SP 800-53.
nit: callout to section number doesn't have a trailing period: "Section 5.5." -> "Section 5.5"